mk/mockery.mk

```diff
@@ -8,4 +8,6 @@ $(GO_OUTPUT)/mockery: ## Install mockery locally on your GO_OUTPUT (./release) d
 
 .PHONY: mock
 mock: $(GO_OUTPUT)/mockery ## Install mockery if it isn't already in ./release directory and regenerate mocks
-	go generate ./...
\ No newline at end of file
+	sed -i '2,$$ s/^/\/\//' ./pkg/dao/registry_mock.go
+	mockery
```

Comment:
wouldn't this call mockery installed outside of the project? i don't have mockery installed locally and this fails for me 😞 maybe it should be `$(GO_OUTPUT)/mockery`?

Reply:
yeah, it definitely should be `$(GO_OUTPUT)/mockery`, didn't realize I was using my locally installed mockery 🤦🏼 
thanks for catching that 👍🏼